PageObject/Config/WebdriverSetup.py

```diff
@@ -2,11 +2,24 @@
 from selenium.webdriver.chrome.options import Options
 
 
-def webdriver_setup():
-    chrome_options = Options()
-    chrome_options.add_experimental_option("detach", True)
-    driver = webdriver.Chrome(options=chrome_options)
-
-    driver.get("https://www.lambdatest.com/")
-    driver.maximize_window()
-    return driver
+# def webdriver_setup():
+#     chrome_options = Options()
+#     chrome_options.add_experimental_option("detach", True)
+#     driver = webdriver.Chrome(options=chrome_options)
+#
+#     driver.get("https://www.lambdatest.com/")
+#     driver.maximize_window()
+#     return driver
+
+class WebdriverSetup:
+    def __init__(self, driver):
+        self.driver = driver
+
+    def webdriver_setup(self):
+        chrome_options = Options()
+        chrome_options.add_experimental_option("detach", True)
+        driver = webdriver.Chrome(options=chrome_options)
```

Comment:
jeśli masz zaincalizowany w init   self.driver = driver


to w metodzie posłuj sie 
```suggestion
       self.driver = webdriver.Chrome(options=chrome_options)
       self.driver.get("https://www.lambdatest.com/")
       self.driver.maximize_window()
       return self.driver
```